src/ILinkGenerator.php

```diff
@@ -12,9 +10,10 @@
 
 	/**
 	 * @param string $dest in format "[[[module:]presenter:]action] [#fragment]"
-	 * @param array $params
+	 * @param mixed[] $params
 	 * @return string
-	 * @throws InvalidLinkException
+	 * @phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint
+	 * @phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingReturnTypeHint
 	 */
 	public function link($dest, array $params = []);
```

Comment:
`string $dest`?

Reply:
I would like, but it's not compatible with Nette LinkGenerator, thus there is `phpcsSupress` ;-)